CMakeLists.txt

```diff
@@ -147,6 +148,27 @@ endif()
 enable_testing()
 add_subdirectory(tests)
 
+# Install the executables and library
+set(LIB_SUFFIX "" CACHE STRING "Suffix for library install directory.")
+install(TARGETS mopac RUNTIME)
+install(TARGETS param RUNTIME)
+if(WIN32)
+  install(TARGETS mopac_core RUNTIME)
+else()
+  set(CMAKE_INSTALL_LIBDIR ${CMAKE_INSTALL_LIBDIR}${LIB_SUFFIX})
+  install(TARGETS mopac_core LIBRARY)
+#  if(LIB_SUFFIX)
+#    install(TARGETS mopac_core DESTINATION lib${LIB_SUFFIX})
+#  else()
+#    install(TARGETS mopac_core DESTINATION lib)
+#  endif()
```

Comment:
Does it make sense to keep this in comments?

Reply:
I intended to remove it. My present understanding is that CMake has intelligent default choices for installation directories that are probably better to modify rather than supersede, but I might be wrong about this.

You added LIB_SUFFIX - what was the intended use? My modifications might not preserve that use, so we might have to tweak this further. The CPack-based installer is a work-in-progress and still isn't fully working as intended.

Reply:
On multiarch platforms there are two library directories: `lib` and `lib64` (Red Hat style) and `lib` and `lib32` (Debian style). You need to be able to define the suffix of the library directory. `LIB_SUFFIX` is the standard way to do this.